src/objc/messages.rs

```diff
@@ -134,7 +152,42 @@ fn objc_msgSend_inner(env: &mut Environment, receiver: id, selector: SEL, super2
 /// Standard variant of `objc_msgSend`. See [objc_msgSend_inner].
 #[allow(non_snake_case)]
 pub(super) fn objc_msgSend(env: &mut Environment, receiver: id, selector: SEL) {
-    objc_msgSend_inner(env, receiver, selector, /* super2: */ None)
+    objc_msgSend_inner(env, receiver, selector, /* super2: */ None, None)
+}
+
+#[repr(C, packed)]
+/// A pointer to this struct replaces the normal receiver parameter for
+/// `objc_msgSend(stret)_debug` and [msg_send] when debug assertions are on.
```

Comment:
That should be `objc_msgSend(_stret)_debug` right?

Also, the comment doesn't seem to match the implementation, isn't it only used for `msg_send_debug`?

Reply:
Oh, also, you could just write `[objc_msgSend_debug], [objc_msgSend_stret_debug] and [msg_send]`.